sdks/python/apache_beam/io/textio.py

```diff
@@ -222,6 +250,7 @@ def __init__(
       strip_trailing_newlines=True,
       coder=coders.StrUtf8Coder(),
       validate=True,
+      skip_header_lines=0,
```

Comment:
This should be passed to _TextSource below.

Please add a test that would have captured this. For example, a pipeline that uses 'DirectRunner'.